pcdsdevices/lodcm.py

```diff
@@ -186,4 +383,276 @@ def remove_dia(self, moved_cb=None, timeout=None, wait=False):
 
         return status
 
+    def _is_tower_1_c(self):
```

Comment:
So i got these guys from the old code, but I am not sure if this is still the case now? (`_is_tower_1_c`, `_is.......`)
These functions will be also used in PR #704 

It seems like right now they all return `False` - if `h1n` position is `C` then `y1`'s position is `Unknown` or some type of combination like this that don't make them `True`

I guess my question here is how do we determine if the tower 1 is `C` or `Si` - is it only based on `h1n`? or are these combinations here still valid?

Reply:
h1n brings the crystal into the beam, moving horizontally (normal to the beam and parallel to the ground). It's also the only axis that can fully clear the crystal assembly from the beam.
y1 brings the crystal into the beam, moving vertically (normal to the beam and normal to the ground)
chi1 rotates the crystal assembly, presumably selecting a crystal on an axis that I don't have a geometric understanding of

So I think this function is correct insofar as:
- If they all agree, we are at that crystal
- If they disagree, we are not at any particular crystal, we're just misaligned

The confusing part is the h1n out clause. I think this is saying "we are technically out, but if we were to insert into the beam, we'd be aligned to the C crystal"

Reply:
Also, tower 2 is never removed so there is no need for the h1n out clause? (guessing)

Reply:
(I have nothing useful to add here; just another request to add the above very useful comment in class docstrings somehow as you see fit...)